### ISSUE TITLE
How to remove

### ISSUE THREAD

**Issue by alberto1el** [kelseyhightower/nocode#1727]
This might be categorized as a bug, please help.

I have not installed any 'nocode' applications on my computer but I have read the documentation and I am aware that 'nocode' applications look something similar to this:

```

```

Well I was just in my cli doing nothing today and I believe that I have completed a 'nocode' application.

I do not know where the 'nocode' application resides in my system or how to remove it.

I am aware that 'nocode' applications bring no security issues or dependencies, but following the instructions I believe I have done many 'nocode' applications I just want to be sure they will do what they are supposed to do and not interfere with the rest of my systems functionality.

**Comment by moizalicious** [comment]
@alberto1el. If you are using Linux, I have a simple fix. Open your terminal and type in the following:
```
```

If you are using Windows then it's a little more complicated. You have to refresh the desktop before typing the above line in the CLI. [Click Here](https://www.techwalla.com/articles/how-to-refresh-your-desktop-in-windows) for a basic tutorial on how to refresh a Windows desktop.

I'm afraid a fix for this has not come out for MacOS yet.

**Comment by alberto1el** [comment]
Thank you @moizalicious for your advice, right now I am not able to log into that computer but will try that as soon as possible and will come back with results.

Thank you also on the windows tut I believe I have summoned more 'nocode' apps into my windows laptop will try that for sure.

**Comment by zx8754** [comment]
> This might be categorized as a bug...

@alberto1el No, it is a feature, no need for any fix. Please close this (non) issue.
